draft-privacypass-rate-limit-tokens.md

```diff
@@ -1003,6 +1003,7 @@ In pseudocode, this is as follows:
 context = concat(token_type,
```

Comment:
nit: Maybe update the "Client produce signatures..." to just explicitly call out signing the entire request, rather than listing the individual bits?

Reply:
Good idea -- done!